cmd/interrupts.go

```diff
@@ -27,7 +27,7 @@ import (
 // ApplicationStopper stops application and performs required cleanup tasks
 type ApplicationStopper func()
 
-// StopOnInterrupts invokes given stopper on SIGTERM and SIGHUP interrupts with additional wait condition
+// StopOnInterruptsConditional invokes given stopper on SIGTERM and SIGHUP interrupts with additional wait condition
```

Comment:
Why this new naming is better? What does "conditional" mean in this context?

Reply:
I just made docblock consistent with code